gno.land/cmd/gnoweb/views/faucet.html

```diff
@@ -7,35 +7,31 @@
   </head>
   <body onload="main()">
     <div id="root">
-      <div id="header">
-        {{ template "header_logo" }}
-        {{ template "header_buttons" }}
-      </div>
-      <div id="home">
-          This is the Gno.land (test) faucet.
+      <div id="header">{{ template "header_logo" }} {{ template "header_buttons" }}</div>
+      <div id="home" class="container">
+        This is the Gno.land (test) faucet. {{ if .Data.captchaSite }}
+        <script src="https://www.google.com/recaptcha/api.js" async defer></script>
+        {{ end }}
+        <script type="text/javascript">
+          function main() {
+              // XXX
+          };
```

Comment:
maybe remove it and remove `onload="main()"` from body?

Reply:
You are right, I wanted to create another PR to do a good cleaning/webperf improvements, since there is a lot of old remaining/duplicate code. But let's improve what we can now. Thank you!